### PR TITLE
adding classification result

### LINKED ISSUE #9: enhancement/adding-classification-data
Users should be able to add classification data to be able to retrain the model

### PR DESCRIPTION
**Pull request by Wisdom0063**
Adding classification data for retraining of model

### CODE REVIEW

File: src/middleware/authenticate.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const { throwError } = require("../lib/errors");
+const { decodeJwt } = require("../lib/utils");
+
+const authenticate = async (req, res, next) => {
+  let token;
+  try {
+    let bearerHeader = req.headers["authorization"];
+    token = bearerHeader.split(" ")[1];
+    if (!bearerHeader) {
+      message = "No authentication token provided";
```
Comment (narteysarso) [inline src/middleware/authenticate.js:12]: Preceding this line with let keyword to declare message variable will resolve Codacy Quality PR

**Review by narteysarso — CHANGES_REQUESTED** [review]
Can we resolve the Codacy quality PR

**Review by narteysarso — APPROVED** [review]
Awesome

**Review by mickeymond — APPROVED** [review]
great

**Pull request merged**